services/def.json

```diff
@@ -0,0 +1,32 @@
+{
+    "roles":
```

Comment:
I tried the namespace and it was not working right.  I left it as the user expects and added a `role` meta tag.